src/main/java/nl/novi/techiteasy/controllers/RemoteControllerController.java

```diff
@@ -0,0 +1,73 @@
+package nl.novi.techiteasy.controllers;
+
+
+import nl.novi.techiteasy.dtos.remotecontroller.RemoteControllerDto;
+import nl.novi.techiteasy.dtos.remotecontroller.RemoteControllerInputDto;
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import nl.novi.techiteasy.exceptions.ValidationException;
+import nl.novi.techiteasy.models.RemoteController;
+import nl.novi.techiteasy.service.RemoteControllerService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+
+import java.net.URI;
+import java.util.List;
+
+import static nl.novi.techiteasy.controllers.ControllerHelper.checkForBindingResult;
+
+@RequestMapping("/remotecontrollers")
+@RestController
+public class RemoteControllerController {
+
+    private final RemoteControllerService remoteControllerService;
+
+    public RemoteControllerController(RemoteControllerService remoteControllerService) {
+        this.remoteControllerService = remoteControllerService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<RemoteControllerDto>> getAllTelevisions(){
+        return ResponseEntity.ok(remoteControllerService.getAllRemoteControllers());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<RemoteControllerDto> getRemoteControllerId(@PathVariable long id){
+        if (id > 0) {
+            RemoteControllerDto rcDto = remoteControllerService.getRemoteControllerId(id);
```

Comment:
Je zou ook op een andere manier kunnen bekijken of het id bestaat, waardoor dit weer anders/beter kan worden opgeschreven worden.